spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -703,21 +701,335 @@ TODO
 All annotations (Except `RegistryType`) have the following fields that correspond to the metadata fields described
 in <<meta-data-def>>.
 
-`String name`::  the name of the metric. If not explicitly given it is determined from the annotated object and the
+`String name`:: Optional. Sets the name of the metric. If not explicitly given it is determined from the annotated object and the
 fully qualified name of the surrounding class, unless `absolute` is true.
-`boolean absolute`:: If `true` use the given name as an absolute name. If `false`, use the given name relative to
-the annotated class
-`String displayName`:: Optionally provide a human readable display name for metadata.
-`String description`:: Optionally provide a description of the metric.
-`MetricUnit unit:: Unit of the metric
-`String[] tags`:: Array of Strings in the `<key>=<value>` format to supply special tags to a metric.
+`boolean absolute`:: If `true` use the given name as an absolute name of the metric. If `false`, use the given name relative to
+the annotated class. Default value is `false`.
+`String displayName`:: Optional. A human readable display name for metadata.
+`String description`:: Optional. A description of the metric.
+`MetricUnit unit`:: Unit of the metric
+`String[] tags`:: Optional. Array of Strings in the `<key>=<value>` format to supply special tags to a metric.
 
 NOTE:: Implementors are encouraged to issue warnings in the server log if metadata is missing. While this is
 less important for description and displayName, it is very important for Units.
 
+[[annotated-naming-convention]]
+==== Annotated Naming Convention
+Annotated metrics are registered into the _application_ `MetricRegistry` with the name based on the annotation's `name` and `absolute` fields.
+
+.Example of annotated metric names
+[source, java]
+----
+package com.example;
```

Comment:
Would we want to add the imports just for completeness?

Reply:
Good suggestion, added it in #47.